src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -338,7 +338,7 @@ private void BuildFallbackBlock(TagHelperContent builder)
 
                 // Build the <meta /> tag that's used to test for the presence of the stylesheet
                 builder
-                    .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" class=\"")
+                    .AppendHtml("<meta property=\"x-stylesheet-fallback-test\" content class=\"")
```

Comment:
@Eilon this is markup that our Tag Helper produces.

@NTaylorMullen we don't put it in the body, we put it next to the `<link />` which should always be in the `<head>`. And who says it can't have a `class` attribute? The report was that it's missing some required attributes, not that it has some illegal attributes.